client/state/selectors/get-jpo-connect.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Get stored home page as was last chosen by user.
```

Comment:
Is this comment correct?

Reply:
Probably copy-pasted from an other selector. I'll get the scoop from them, fix and merge!

Reply:
Please use
`Get object with Jetpack connection properties`

Reply:
Elio, is it a string or an object? In the schema, we defined it as a string.

Reply:
@roccotripaldi yes this is an object with these items:
```
queryObject: {
	client_id: string
	redirect_uri: string
	state: string
	scope: string
	user_email: string
	user_login: string
	jp_version: string
	secret: string
	locale: string
	blogname: string
	site_url: string
	home_url: string
	site_icon: string
	site_lang: string
	_ui: string
	_ut: string
	onboarding: string
	calypso_env: string
	from: string
	_wp_nonce: string
	redirect_after_auth: string
	site: string
}
isAuthorizing: bool
authorizeSuccess: bool
authorizeError: bool
timestamp: number
userAlreadyConnected: bool
siteSlug: string
```